src/client.ts

```diff
@@ -160,6 +162,7 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
     this._config.logger = this.logger;
 
     this.media = new SdkMedia(this);
+    this.headset = new SdkHeadset(this);
```

Comment:
Do we want to have a config option to opt-out of headsets? I doubt there is much of a performance hit to just spin this up and not use any of the implementations, but I could be wrong.

Reply:
I'm not opposed to that but not sure how necessary that would be since most users would want call controls from their headsets right?